plugins/src/main/java/org/smartregister/fhir/gateway/plugins/SyncAccessDecision.java

```diff
@@ -49,14 +50,15 @@ public class SyncAccessDecision implements AccessDecision {
     private static final Logger logger = LoggerFactory.getLogger(SyncAccessDecision.class);
     private final String syncStrategy;
     private final boolean accessGranted;
-    private Map<String, List<String>> syncStrategyIdsMap;
+    private final Map<String, List<String>> syncStrategyIdsMap;
     private final List<String> roles;
     private IgnoredResourcesConfig config;
     private final String keycloakUUID;
     private final Gson gson = new Gson();
     private FhirContext fhirR4Context;
     private final IParser fhirR4JsonParser;
     private IGenericClient fhirR4Client;
+    private static final int REL_LOCATION_CHUNKSIZE = 20;
```

Comment:
Is this configurable?

Reply:
No it isn't and I don't think it should be at the moment since this is where the Large URI Exception stems from. We want it to be the least highest possible value.

Reply:
Makes sense